apps/zbugs/server/notify.ts

```diff
@@ -71,40 +64,70 @@ export async function notify(
   const modifierUser = await tx.query.user.where('id', modifierUserID).one();
   assert(modifierUser);
 
-  const recipients = await gatherRecipients(
-    // TODO: we shouldn't have to assert and cast this given this is called from `server-mutators`.
-    tx as ServerTransaction<Schema, postgres.TransactionSql>,
-    issueID,
-    isAssigneeChange,
-    previousAssigneeID,
-  );
+  const recipientEmails = await gatherRecipients(tx, issueID, modifierUserID);
 
   // If no recipients, skip notification
-  if (recipients.length === 0) {
+  if (recipientEmails.length === 0) {
     console.log('No recipients for notification', issueID);
     return;
   }
 
+  if (issue.shortID === null) {
+    console.log('No short ID for issue', issueID);
```

Comment:
This should not be possible. By the time we get to this code it should have been filled in. Maybe `assertNotNull()` instead.